app/models/application_form.rb

```diff
@@ -130,6 +130,7 @@ class ApplicationForm < ApplicationRecord
          awarded: "awarded",
          declined: "declined",
          potential_duplicate_in_dqt: "potential_duplicate_in_dqt",
+         assessment_in_progress: "assessment_in_progress",
```

Comment:
Just a minor point, but could we put this next to `initial_assessment` as the order of these vaguely matches the order the application would go through.